### DISCUSSION TITLE
Some Feature requests

### DISCUSSION THREAD

**Discussion by blueicehaller** [DrMichael/FritzCall#9] [Ideas]
Hi @DrMichael,

thank you for enabling "Discussions" (#7) .

May I request some features (#8)?
I don't know what's possible?
Should I create separate discussions?

- If `Rufumleitung` (call forwarding) is active, display the target number(s) 
- If `Rufumleitung` (call forwarding) is active, display and when (`Verzögert`, `Sofort`)
- Enable / disable `Rufumleitung`
- Configure `Rufumleitung`: Change from `Verzögert` to `Sofort`
- Enable / disable `DECT` with blue button

Thank you in advance.

**Comment by DrMichael** [comment]
Ugh, sorry for the very late answer: no, that information is not delivered in the data, the plugin gets.
